tests/cases/Issue415.tsx

```diff
@@ -0,0 +1,7 @@
+(<View />);
```

Comment:
Add below comment on the first line so that the baseline would be generated only against React grammar
```
// @onlyOwnGrammar - As this has jsx
```